templates/zerver/help/desktop-app-install-guide.md

```diff
@@ -8,7 +8,7 @@ To install the latest stable release (recommended for most users),
 find your operating system below.  If you're interested in an early
 look at the newest features, consider the [beta releases](#beta-releases).
 
-## Installing on macOS
+## [Installing on Mac](#installing-on-mac)
```

Comment:
Do you need these to be markdown links to themselves?

Reply:
Yup. `## Installing on macOS` on GitHub markdown works but on readthedocs it doesn't so we need to link them like this only.

Reply:
huh, I thought this worked in RTD without anything special...